docs/discovery/webfinger.md

```diff
@@ -6,7 +6,7 @@ diaspora\* uses an old draft of [WebFinger][webfinger-draft] to discover users f
 
 {% include warning_box.html
    title="Old WebFinger"
-   content="<p>diaspora* doesn't yet support the RFC 7033 WebFinger!</p>"
+   content="<p>diaspora* doesn't yet fully support the RFC 7033 WebFinger!</p>"
```

Comment:
There is a line above in the doc that says `diaspora* uses an old draft of WebFinger to discover users from other pods.`. This is no longer true. We can change that to `diaspora* supports RFC 7033 WebFinger to discover users from other pods.`. Here we have this "Old WebFinger" notice. If we have the above text fixed, then we can make more sense to change the context of the box to something like `diaspora* below version 0.7.0.0 supports only Old draft WebFinger. RFC 7033 WebFinger is supported from 0.7.0.0 and higher. This document focuses on old WebFinger.`.

Also we have "Additional information and specifications" box at the page footer. We could actually add RFC 7033 WebFinger specification link there.